Docs/sphinx/DeveloperGuide.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: rst
+
+
+Developer Guidelines
+====================
+
+The following developer guidelines apply to all code that is to be committed to the `Pele` suite. 
+
+Before adding files for a commit to be pushed to a branch of `PelePhysics`, first run the following formatting tools depending on the type of code:
+
+
+C++ Code
+--------
+All C++ code should be processed by the Clang formatter prior to being added for commit.
+
+Run ``clang-format``::
+
+    clang-format -i .
+
+Will apply all of the correct formatting and make the changes directly to the files.
+
+
+Python
+------
+
+The tools necessary to format the `Python` code are maintained through `poetry`.
```

Comment:
Do you want to specify that this is really for `ceptr` only?

Reply:
missed this one...updating now